ddtrace/opentelemetry/span.go

```diff
@@ -10,22 +10,21 @@ import (
 	"strconv"
 	"strings"
 
-	"gopkg.in/DataDog/dd-trace-go.v1/internal/log"
-
 	"go.opentelemetry.io/otel/attribute"
 	otelcodes "go.opentelemetry.io/otel/codes"
 	oteltrace "go.opentelemetry.io/otel/trace"
 
-	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/log"
 )
 
 var _ oteltrace.Span = (*span)(nil)
 
 type span struct {
-	ddtrace.Span
-	finished bool
+	tracer.Span
```

Comment:
Why change the type?

Reply:
It seems like in all the code examples, we use trace.StartOption,  tracer.Span, and so on. I wanted to preserve that